tests/new_upgrades/test_satellitesync.py

```diff
@@ -0,0 +1,113 @@
+"""Test for Inter Satellite Sync related Upgrade Scenario's
+
+:Requirement: UpgradedSatellite
+
+:CaseAutomation: Automated
+
+:CaseComponent: InterSatelliteSync
+
+:Team: Phoenix-subscriptions
+
+:CaseImportance: High
+
+"""
+
+from box import Box
+from fauxfactory import gen_alpha
+import pytest
+
+from robottelo.config import settings
+from robottelo.constants import PULP_EXPORT_DIR
+from robottelo.utils.shared_resource import SharedResource
+
+
+@pytest.fixture
+def version_cv_export_import_setup(content_upgrade_shared_satellite, upgrade_action):
+    """Before upgrade, create the content view and publish, and promote it.
+
+    :steps:
+        1. Create a ContentView
+        2. Publish and promote the Content View
+        3. Check the package count of promoted content view.
+
+    :expectedresults: Before the upgrade, Content view published and promoted, and package
+        count should be greater than 0.
+    """
+    target_sat = content_upgrade_shared_satellite
+    with SharedResource(target_sat.hostname, upgrade_action, target_sat=target_sat) as sat_upgrade:
+        test_name = f'sat_sync_upgrade_{gen_alpha()}'
+        org = target_sat.api.Organization(name=f'{test_name}_org').create()
+        product = target_sat.api.Product(name=f'{test_name}_prod', organization=org.id).create()
+        repo = target_sat.api.Repository(
+            name=f'{test_name}_repo',
+            content_type='yum',
+            product=product,
+            download_policy='immediate',
+            url=settings.repos.yum_1.url,
+        ).create()
+        repo.sync()
+        cv = target_sat.publish_content_view(org, repo, f'{test_name}_content_view')
+        assert cv.version[0].read().package_count > 0
+        test_data = Box(
+            {
+                "test_name": test_name,
+                "satellite": target_sat,
+                "org": org,
+                "content_view": cv,
+            }
+        )
+        sat_upgrade.ready()
+        target_sat._session = None
+        yield test_data
+
+
+@pytest.mark.content_upgrades
+def test_post_version_cv_export_import(version_cv_export_import_setup):
+    """After upgrade, content view version import and export works on the existing content
+     view(that we created before the upgrade).
+    :id: postupgrade-f19e4928-94db-4df6-8ce8-b5e4afe34258
+    :parametrized: yes
+    :steps:
+        1: Export the existing content-view version.
+        2: Import the existing content-view version.
+        3: Delete the imported and exported content-vew, product, repo and organization.
```

Comment:
Please update the steps if we decide not to finalize the CV, prod, and Org as we did originally (I don't mind keeping them).
```suggestion
        2: Import the exported content-view version.
        3: Delete the imported and exported content-view from LCE.
```